config/coreboot-omnigul.config

```diff
@@ -0,0 +1,1473 @@
+CONFIG_VPD_FMAP_NAME="RO_VPD"
+CONFIG_VPD_FMAP_SIZE=0x4000
+CONFIG_DRIVERS_GENERIC_ALC1015=y
+# CONFIG_DRIVERS_GENERIC_CBFS_SERIAL is not set
+# CONFIG_DRIVERS_GENERIC_CBFS_UUID is not set
+CONFIG_DRIVERS_GENERIC_GPIO_KEYS=y
+CONFIG_DRIVERS_GENERIC_MAX98357A=y
+# CONFIG_DRIVERS_GENESYSLOGIC_GL9750 is not set
+# CONFIG_DRIVERS_GENESYSLOGIC_GL9755 is not set
+# CONFIG_DRIVERS_GENESYSLOGIC_GL9763E is not set
+CONFIG_DRIVERS_GFX_GENERIC=y
+CONFIG_DRIVERS_I2C_DESIGNWARE=y
+CONFIG_DRIVERS_I2C_GENERIC=y
+CONFIG_DRIVERS_I2C_HID=y
+# CONFIG_DRIVERS_I2C_MAX98396 is not set
+CONFIG_DRIVERS_I2C_NAU8825=y
+CONFIG_I2C_TPM=y
+CONFIG_DRIVER_TIS_DEFAULT=y
+CONFIG_DRIVER_I2C_TPM_ACPI=y
+# CONFIG_DRIVER_TPM_DISPLAY_TIS_BYTES is not set
+CONFIG_DRIVERS_INTEL_DPTF=y
+CONFIG_DRIVERS_INTEL_DPTF_SUPPORTS_TPCH=y
+CONFIG_FSP_USE_REPO=y
+# CONFIG_DISPLAY_HOBS is not set
+# CONFIG_DISPLAY_UPD_DATA is not set
+# CONFIG_BMP_LOGO is not set
+CONFIG_PLATFORM_USES_FSP2_0=y
+CONFIG_PLATFORM_USES_FSP2_1=y
+CONFIG_PLATFORM_USES_FSP2_2=y
+CONFIG_PLATFORM_USES_FSP2_3=y
+CONFIG_PLATFORM_USES_FSP2_X86_32=y
+CONFIG_HAVE_INTEL_FSP_REPO=y
+CONFIG_ADD_FSP_BINARIES=y
+CONFIG_FSP_T_LOCATION=0xfffe0000
+CONFIG_FSP_S_CBFS="fsps.bin"
+CONFIG_FSP_M_CBFS="fspm.bin"
+CONFIG_FSP_FULL_FD=y
+CONFIG_FSP_T_RESERVED_SIZE=0x0
+CONFIG_FSP_M_XIP=y
+CONFIG_FSP_USES_CB_STACK=y
+CONFIG_HAVE_FSP_LOGO_SUPPORT=y
+CONFIG_FSP_COMPRESS_FSP_S_LZ4=y
+CONFIG_FSP_STATUS_GLOBAL_RESET_REQUIRED_3=y
+CONFIG_FSP_STATUS_GLOBAL_RESET=0x40000003
+CONFIG_SOC_INTEL_COMMON_FSP_RESET=y
+CONFIG_FSPS_HAS_ARCH_UPD=y
+CONFIG_FSPS_USE_MULTI_PHASE_INIT=y
+CONFIG_FSP_USES_CB_DEBUG_EVENT_HANDLER=y
+# CONFIG_DISPLAY_FSP_TIMESTAMPS is not set
+CONFIG_FSP_ENABLE_SERIAL_DEBUG=y
+CONFIG_FSP_MULTIPHASE_SI_INIT_RETURN_BROKEN=y
+CONFIG_INTEL_GMA_ACPI=y
+CONFIG_VBT_CBFS_COMPRESSION_LZMA=y
+# CONFIG_VBT_CBFS_COMPRESSION_LZ4 is not set
+# CONFIG_VBT_CBFS_COMPRESSION_NONE is not set
+CONFIG_VBT_CBFS_COMPRESSION_ALGORITHM="lzma"
+CONFIG_INTEL_GMA_OPREGION_2_1=y
+CONFIG_INTEL_GMA_VERSION_2=y
+CONFIG_DRIVERS_INTEL_ISH=y
+CONFIG_DRIVERS_INTEL_PMC=y
+CONFIG_DRIVERS_INTEL_SOUNDWIRE=y
+# CONFIG_DRIVERS_NXP_UWB_SR1XX is not set
+# CONFIG_DRIVERS_PS2_KEYBOARD is not set
+CONFIG_DRIVERS_MC146818=y
+CONFIG_USE_PC_CMOS_ALTCENTURY=y
+CONFIG_PC_CMOS_BASE_PORT_BANK0=0x70
+# CONFIG_DRIVERS_SIL_3114 is not set
+CONFIG_DRIVERS_SOUNDWIRE_ALC5682=y
+CONFIG_DRIVERS_SOUNDWIRE_MAX98373=y
+CONFIG_DRIVERS_SPI_ACPI=y
+CONFIG_DRIVERS_USB_ACPI=y
+CONFIG_DRIVERS_WIFI_GENERIC=y
+CONFIG_DRIVERS_WWAN_FM350GL=y
+CONFIG_MP_SERVICES_PPI=y
+CONFIG_MP_SERVICES_PPI_V2=y
+CONFIG_DRIVERS_INTEL_USB4_RETIMER=y
+# end of Generic Drivers
+
+#
+# Security
+#
+
+#
+# CBFS verification
+#
+# CONFIG_CBFS_VERIFICATION is not set
+# end of CBFS verification
+
+#
+# Verified Boot (vboot)
+#
+CONFIG_VBOOT_LIB=y
+# end of Verified Boot (vboot)
+
+CONFIG_TPM_GOOGLE=y
+CONFIG_TPM_GOOGLE_CR50=y
+CONFIG_TPM_GOOGLE_IMMEDIATELY_COMMIT_FW_SECDATA=y
+CONFIG_GOOGLE_TPM_IRQ_TIMEOUT_MS=100
```

Comment:
Might be too small. Is there timeouts in dmesg?

Reply:
Not that I've seen. I can try modifying that.